libs/initScripts/local.js

```diff
@@ -68,14 +68,27 @@ if (args.length < 3) {
   throwArgError()
 }
 
+const getEnvConfigPathsForService = async ({ workspace, serviceCount }) => {
+  const tmpDir = `${workspace}/tmp${serviceCount}`
+  const cleanupNeeded = await fs.existsSync(tmpDir)
+  if (cleanupNeeded) {
+    await fs.rmdirSync(tmpDir, { force: true, recursive: true })
+  }
+  const templatePath = `${workspace}/commonEnv.sh`
+  const envPath = `${workspace}/shellEnv${serviceCount}.sh`
+  const writePath = `${tmpDir}/shellEnv.sh`
+  await fs.mkdirSync(tmpDir)
```

Comment:
I think instead of having a separate tmpDir for each instance we could have one and name the files something like tmpEnv#.sh

Reply:
Yep! That can make git diff kind of tricky.